.env.demo

```diff
@@ -128,3 +128,5 @@ AGENT_PROTOCOL=http
 OOB_BATCH_SIZE=10
 MAX_ORG_LIMIT=10
 FIDO_API_ENDPOINT=http://localhost:8000
+
+IS_ECOSYSTEM_ENABLE=true
```

Comment:
Can you make this as false as we won't be needing to have ecosystem enables

Reply:
Resolved